### DISCUSSION TITLE
Manual Installation on Windows without WSL

### DISCUSSION THREAD

**Discussion by ninihub1** [spiral-project/ihatemoney#1314] [Q&A] [answered]
Hello,

I am trying to set up ihatemoney on my Windows machine and I'm encountering some issues. Here are the steps I followed and the problems I'm facing:

Generate Configuration File:
ihatemoney generate-config ihatemoney.cfg > ihatemoney.cfg
This command runs without any visible errors.

Set Environment Variable:
export IHATEMONEY_SETTINGS_FILE_PATH=$PWD/ihatemoney.cfg
This seems to work fine.

Upgrade Database:
ihatemoney db upgrade head

I receive the following error:
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file

I checked and the database file ihatemoney.sqlite does not exist.

**Comment by almet** [comment]
Hi, for windows, the recommended way of installing is via WSL2. Is it what you did?

**Reply by ninihub1** [reply]
No, I used the manual installation instruction that are in the Online Documentation.

**Reply by almet** [reply, marked as answer]
I understand. It's not currently clearly stated in our documentation. If that's possible for you, I recommend you to install inside WSL, as other have made it work in the past.

I'm closing this, but feel free to reopen if there is a need.

**Comment by ninihub1** [comment]
[like]  Blony Maunela reacted to your message:
________________________________
